README.md

```diff
@@ -52,9 +52,24 @@ Participation in the ZClassic project is subject to a
 
 Build ZClassic along with most dependencies from source by running:
 
+
+ Get dependencies
+```{r, engine='bash'}
+sudo apt-get install \
+      build-essential pkg-config libc6-dev m4 g++-multilib \
+      autoconf libtool ncurses-dev unzip git python \
+      zlib1g-dev wget bsdmainutils automake
 ```
-./zcutil/build.sh -j$(nproc)
-```
+
+ Install
+```{r, engine='bash'}
+# Build
+./zcutil/build.sh -j$(nproc)	./zcutil/build.sh -j$(nproc)
```

Comment:
duplication in line

Reply:
removed duplicate in line 67, sorry about that.